### ISSUE TITLE
Cost Issues with One-Time-Use Summon Artifacts 

### ISSUE THREAD

**Issue by ricegf** [ricegf/elven-fire#32]
While working on Tep, I realized a possible hole in the summoning rules with respect to gems, scrolls and potions.

The IIQ for Summon affects 2 things - the fatigue cost of summoning and (for called or cloned) sustaining the summoned creature, and the distance from the summoner that the creature appears. Since gems are self-powered, the former doesn't matter, and since the gem can be thrown, the latter isn't terribly important, either.

Thus, a Summon 1 and Summon 2 gem should cost exactly the same, and a Summon 5 gem only slightly more. This applies to a somewhat lesser extent to scrolls and ability potions (since distance is always from the summoner in those cases).

One possible solution to making IIQ more relevant to gems, scrolls and potions of Summoning would be to cap the IQ applied based on IIQ for those artifacts only. So, regardless of the character's IQ, using gem, scroll or potion of Summoning IIQ 1 would be at IQ 10 or the character's IQ, whichever was lower. For IIQ 2 use 12, for IIQ 3 use 15, for IIQ 4 use 21, and no limits on IIQ 5.

I'm open to other options as well, including simply declaring a special pricing method for one-time-use Summoning artifacts if you think that would be better.

**Comment by sarinilla** [comment]
Hmm.  I am inclined to believe "no harm, no foul" on this one.  Why complicate the rules unnecessarily?

**Comment by JaceR** [comment]
I tend to agree with Rini on that. Honestly, the cost of a low IIQ gem or scroll isn't **that** expensive, anyway.

**Comment by Grendus** [comment]
The only issue I see with potions, gems, and scrolls of summon is that the power limit is based on IQ, not summon skill. Thus, a scroll or gem of summon 1 is identical to summon 5 in terms of what you can summon since they're self powered. In fact, since you don't have to roll when using a gem, you can theoretically summon an unlimited power with a gem of summon 1. While no DM would allow this, it is RAW legal.

What I would suggest would be to limit the total power summoned based on the IIQ or the IQ (say, IIQ_5 summon, IIQ_10 illusion/image, though the numbers can and probably should be tweaked before being set as a rule), whichever limit is lower, with no IIQ based limit at IIQ 5. That way, IIQ 2 summon is an improvement in items, and the loophole is closed before it can be abused.
